selfservice/strategy/code/strategy_login.go

```diff
@@ -194,22 +197,37 @@ func (s *Strategy) loginSendEmail(ctx context.Context, w http.ResponseWriter, r
 
 	p.Identifier = maybeNormalizeEmail(p.Identifier)
 
-	// Step 1: Get the identity
-	i, _, err := s.findIdentityByIdentifier(ctx, p.Identifier)
-	if err != nil {
-		return err
+	var addresses []Address
+	var i *identity.Identity
+	if f.RequestedAAL > identity.AuthenticatorAssuranceLevel1 {
+		address, found := lo.Find(sess.Identity.VerifiableAddresses, func(va identity.VerifiableAddress) bool {
```

Comment:
Why aren't we sending it to the value of `p.Identifier` similar to how we do it for login/registration? I fear that if verified address is not set correctly, this flow won't work and it will not be clear why (just says "unknown address").

Reply:
We need to find out how to send the code to the user, which we only know through the verifiable address (`.Via`). 

But I don't really understand why we didn't do this for login and registration as well, in the first place.

Reply:
I thought about this some more, and came to the conclusion that the best way to do this is using verifiable addresses, and that we should've done that for the normal login via code as well: With verifiable addresses we already established a pattern of "an OOB address, that can receive _something_". Having the code credential define this path in the identity schema again, essentially duplicates this for no (or very little) benefit. (Side note: this would've allowed us to build configs like `"require verified address for login via code"` way easier.)

However, repeating this mistake with MFA isn't necessary, IMO. If we were to use the code credential defined way of delivering the OTP for MFA, we're just delegating this config to the code login (which MFA is sort of, but maybe not obviously so). Verifiable addresses fit this criterion better, IMO.